src/Tizen.Content.MediaContent/Tizen.Content.MediaContent/FaceInfoCommand.cs

```diff
@@ -140,5 +140,80 @@ public bool UpdateTag(string faceInfoId, string tag)
                 Interop.Face.Destroy(handle);
             }
         }
+
+        /// <summary>
+        /// Inserts a new face information to the database with the specified media ID, area, orientation.
+        /// </summary>
+        /// <privilege>http://tizen.org/privilege/content.write</privilege>
+        /// <param name="mediaId">The media ID to be associated with.</param>
+        /// <param name="area">The region of face in the media.</param>
+        /// <param name="orientation">The orientation of specified media.</param>
+        /// <returns>The <see cref="FaceInfo"/> containing the results.</returns>
+        /// <exception cref="InvalidOperationException">The <see cref="MediaDatabase"/> is disconnected.</exception>
+        /// <exception cref="ObjectDisposedException">The <see cref="MediaDatabase"/> has already been disposed of.</exception>
+        /// <exception cref="MediaDatabaseException">An error occurred while executing the command.</exception>
+        /// <exception cref="ArgumentNullException"><paramref name="mediaId"/> is null. </exception>
+        /// <exception cref="ArgumentException">
+        ///     <paramref name="mediaId"/> is a zero-length string, contains only white space.<br/>
+        ///     -or-<br/>
+        ///     <paramref name="orientation"/> is not valid enumeration.
+        /// </exception>
+        /// <exception cref="UnauthorizedAccessException">The caller has no required privilege.</exception>
+        /// <since_tizen> 6 </since_tizen>
+        public FaceInfo Insert(string mediaId, Rectangle area, Orientation orientation)
+        {
+            return Insert(mediaId, area, orientation, null);
+        }
+
+        /// <summary>
+        /// Inserts a new face information to the database with the specified media ID, area, orientation, and tag.
```

Comment:
Suggestion:
/// Inserts new face information to the database with the specified media ID, area, orientation, and tag.


Other rephrases are same as above.